### PR TITLE
Development

### PR DESCRIPTION
**Pull request by ultraviolet-jordan**

### CODE REVIEW

File: cache/src/main/java/com/osrsd/cache/Library.java [hunk 1]
```diff
@@ -17,16 +21,32 @@ public Library(final String path) {
...
+     *
+     * @param indexId The indexId to read from the cache library.
+     * @return The index.
+     */
     @Synchronized
-    public byte[] data(final int index, final int archive, final int fileId) {
-        return cacheLibrary.data(index, archive, fileId);
+    public Index index(final int indexId) {
+        Index index = cacheLibrary.index(indexId);
+        index.cache();
```
Comment (tyler27) [inline cache/src/main/java/com/osrsd/cache/Library.java:48]: What does index.cache do?

File: app/src/main/java/com/osrsd/App.java [hunk 2]
```diff
@@ -55,7 +61,10 @@ private static void init() {
                 new PrintVarbits(library),
                 new PrintSoundEffects(library),
```
Comment (tyler27) [inline app/src/main/java/com/osrsd/App.java:62]: We should make this configurable in the future, so we can run App with arguments like:
`print --varbits --items --soundeffects --objects`

**Review by tyler27 — APPROVED** [review]
Approved, but left a comment.

File: app/src/main/java/com/osrsd/App.java (hunk 2 above)
Comment (ultraviolet-jordan) [reply app/src/main/java/com/osrsd/App.java:62]: Yes that is a good idea.

**Pull request merged**